examples/hello_world.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Example OpenHTF test logic.
+
+Run with (your virtualenv must be activated first):
+python hello_world.py
+"""
+
+# Import this output mechanism as it's the specific one we want to use.
+from openhtf.io.output import json_factory
+
+# Import a handful of useful names.  If you're worried about polluting
+# your namespace, you can manually import just the things you want, this
+# is just a convenience.  See names.py for an exhaustive list.
+from openhtf.names import *
+
+# TODO(madsci): Clean up conf so we don't need to do this.
+from openhtf import conf
+
+
+# The @measures annotation notifies the OpenHTF framework that this test
+# phase will be taking a measurement that we'd like to call
+# 'hello_world_measurement'.  Measurements can be accessed and set via
+# the 'test' object, always passed as the first argument to test phases.
+@measures(Measurement('hello_world_measurement'))
+def hello_world(test):
+  """A hello world test phase."""
+  # At the heart of an OpenHTF test script are the test phases, such as
+  # this one.  Any callable can be used as a test phase, so long as it
+  # accepts a single argument that is the 'test' object.  This test object
+  # is how you will interact with the OpenHTF test framework once a test is
+  # running.  See other examples for more complex cases, but here is a good
+  # example of the sort of minimal functionality you're likely to use.
+
+  # The test.logger attribute is a Python logger instance that is configured
+  # to log to the test record we will output at the end of the test.  This
+  # is the recommended way to do any logging within test phases (this is also
+  # how to get logs to show up in the frontend).
+  test.logger.info('Hello World!')
+
+  # As described above, this is how measurements are set.  The value that has
+  # been set can also be accessed, but can only be set once.
```

Comment:
Hm, whoops.  i think this will change when we validation happens at set time to avoid re-validation and weird frontend behavior (once-failing test becoming passing), but i'll update the comment to be accurate for now.